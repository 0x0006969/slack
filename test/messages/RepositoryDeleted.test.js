const RepositoryDeleted = require('../../lib/messages/repository-deleted');

const repositoryDeletedEvent = require('../fixtures/webhooks/repository.deleted.json');

describe('Repository deleted event rendering', () => {
  test('works', async () => {
    expect(new RepositoryDeleted(repositoryDeletedEvent).toJSON()).toMatchSnapshot();
  });
});
