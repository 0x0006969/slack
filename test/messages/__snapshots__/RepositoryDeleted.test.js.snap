// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repository deleted event rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "#24292f",
      "fallback": "Repository deleted by wilhelmklopp",
      "footer": "<https://github.com/github-slack-test-org/i-will-be-deleted|github-slack-test-org/i-will-be-deleted>",
      "footer_icon": "https://assets-cdn.github.com/favicon.ico",
      "pretext": "Repository deleted by wilhelmklopp",
      "text": "The subscription to this repository has been disabled.",
    },
  ],
}
`;
